b64ct/src/lib.rs

```diff
@@ -70,10 +73,13 @@ pub fn encode<'a>(src: &[u8], dst: &'a mut [u8]) -> Result<&'a str, InvalidLengt
 }
 
 /// Encode the input byte slice as a "B64"-encoded [`String`].
+///
+/// # Panics
+/// If `input` length is greater than `usize::MAX/4`.
 #[cfg(feature = "alloc")]
 #[cfg_attr(docsrs, doc(cfg(feature = "alloc")))]
 pub fn encode_string(input: &[u8]) -> String {
-    let elen = encoded_len(input);
+    let elen = encoded_len_inner(input.len()).expect("input is too big");
```

Comment:
I'd be ok with making this API fallible for this case

Reply:
I am not sure if it's worth the trouble. In the first place it's a convenience method and probability of encountering values larger than 1 GiB is extremely small. If someone wants to be strictly panic-free, they always can use the underlying function directly.